test/e2e/cross_os/test_job_submissions.py

```diff
@@ -103,7 +102,11 @@ def test_job_reports_failed_session_action(
                                 {
                                     "name": "attr.worker.os.family",
                                     "allOf": [os.environ["OPERATING_SYSTEM"]],
-                                }
+                                },
+                                {
+                                    "name": "attr.TestFleetType",
+                                    "anyOf": ["FleetforSession"],
+                                },
```

Comment:
Instead of hardcoding the attribute into the job templates everywhere we need it, how about placing this in a common function/variable and just referencing that instead?